src/test/components/AppLocalizationProvider.test.js

```diff
@@ -134,4 +148,53 @@ describe('AppLocalizationProvider', () => {
     // $FlowExpectError Our version of flow doesn't know about document.dir
     expect(document.dir).toBe('rtl');
   });
+
+  it('fetches FTL strings for all available locales, but not others', async () => {
+    const { store, translatedText } = setup({
+      // At the time of this  writing, "de" is available but "fr-QC" isn't.
+      // Please replace the languages in this test if this changes in the future.
+      languages: ['de', 'fr-QC'],
+    });
+    render(
+      <Provider store={store}>
+        <AppLocalizationProvider>
+          <Localized id="test-id">
+            <span>Fallback String</span>
+          </Localized>
+        </AppLocalizationProvider>
+      </Provider>
+    );
+
+    expect(await screen.findByText(translatedText('de'))).toBeInTheDocument();
+    expect(document.documentElement).toHaveAttribute('lang', 'de');
+    expect(window.fetch).toBeCalledWith('/locales/de/app.ftl');
+    expect(window.fetch).toBeCalledWith('/locales/en-US/app.ftl');
+    expect(window.fetch).toBeCalledTimes(2);
+  });
+
+  it('Falls back properly on en-US if the primary locale lacks a string', async () => {
```

Comment:
super nit: I think we are always starting with lowercase in the `it` blocks, so it completes to a sentence with describe strings. Falls -> falls